### DISCUSSION TITLE
jellyfin有点呆，没法识别媒体库二级分类，大家有解决办法吗

### DISCUSSION THREAD

**Discussion by cng10001** [jxxghp/MoviePilot#846] [问题咨询]
我以为是外国佬软件，所以还试了英文二级分类，还是不行哎。当然可以手动选到二级分类文件夹，但是如果有十几个二级目录是不是要手动添加十几次媒体库啊，是不是有点傻-  -

> ![二级分类](https://github.com/jxxghp/MoviePilot/assets/115156545/986faec3-f6dc-4082-86c6-a648af7ec459)
如图，anime文件夹给我识别成废材王子
动漫文件夹识别成元尊 动漫态  这么不智能的嘛

**Comment by serenity666** [comment]
![Screenshot_2023-10-19-02-42-45-653_org.jellyfin.mobile.jpg](https://github.com/jxxghp/MoviePilot/assets/64485323/14fbee9d-acc2-45f1-88ad-96395a732b35)

把文件夹显示打开就好了

**Reply by cng10001** [reply]
额，开不开好像没有什么区别，都是这样显示的，最好还是用mp这样的第三方软件刮削好了再就分到三文件夹（电视、电影、电视/动漫）一股脑添加进来就好了。
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/d715bea8-887a-4e3c-a5fd-aa700e9224f2)

**Comment by Shurelol** [comment]
添加媒体库添加路径只能到具体媒体的上一级

比如D:/media/tv/anime/来自深渊/来自深渊-S01E01.mkv

只能添加anime这个文件夹，而不能更上级比如tv

**Reply by cng10001** [reply]
是这样的 二级分类对我没有用，所以我把媒体库的二级分类关掉了，就不用一个个添加了。

**Comment by happyTonakai** [comment]
楼主解决了吗？我电影/剧集/动漫下面都有各自的子文件夹，结果剧集和动漫都会显示二级分类，电影不会，感觉好迷……

**Reply by cng10001** [reply]
我是不用二级分类了，感觉也没什么用，你不能识别的话你就一个个添加媒体库目录吧

**Comment by werran2** [comment]
> 楼主解决了吗？我电影/剧集/动漫下面都有各自的子文件夹，结果剧集和动漫都会显示二级分类，电影不会，感觉好迷……

kodi也是这样，电影会无视二级分类，其他的都不行

**Comment by hinsky** [comment]
添加媒体库，选择 混合电影和电视剧 这个内容类型
![image](https://github.com/jxxghp/MoviePilot/assets/59257370/8a1fa067-5aae-422f-bb72-6e249f30fdf4)

扫描库出文件夹结构后，编辑元数据
把每个最小分类的剧集的文件夹类似设置成”节目”，更上级的目录设置成“继承”就可以了
![image](https://github.com/jxxghp/MoviePilot/assets/59257370/62c17c54-5107-4362-98b1-fd9bda5d5847)

**Reply by 532763233** [reply]
ty

**Reply by TPAXcc** [reply]
不是 最新版9.9没这选项了啊

**Comment by cddjr** [comment]
所以我直接关闭了二级分类，个人觉得既然交给jellyfin这类软件做影视墙，也就没有二级分类的必要了，那个只是方便用文件浏览器访问、检索。

**Reply by cng10001** [reply]
对的
但是有些人就喜欢把日韩电影，欧美电影，华语电影……分得很清楚这种偏本地管理的方式。可能他们最后还是喜欢打开文件夹去找片子~

**Reply by deity9580** [reply]
那是你的片子太少
